tensorflow/contrib/tensorrt/kernels/trt_engine_op.cc

```diff
@@ -436,11 +434,9 @@ EngineCtxPair& TRTEngineOp::GetEngine(
   const int batch_size = input_shapes[0].dim_size(0);
 
   // Get engine cache
-  auto trt_rm = TRTResourceManager::instance();
-  auto res_mgr = trt_rm->getManager("TRTEngineCache");
   TRTEngineCacheResource* cache_res = nullptr;
-  auto status = res_mgr->LookupOrCreate(
-      funcdef_name_, "EngineCache", &cache_res,
+  auto status = ctx->resource_manager()->LookupOrCreate(
+      funcdef_name_, "TRTEngineCache", &cache_res,
```

Comment:
nit: how about switch the position of `funcdef_name_` and `"TRTEngineCache"`, so that `"TRTEngineCache"` becomes the name of the container and `funcdef_name_` becomes the name of the resource inside that container.